docs/usage/customize.md

```diff
@@ -77,6 +77,20 @@ serializes POST bodies.
 
 The default encoder is `Faraday::NestedParamsEncoder`.
 
+### Order of parameters
+
+By default, paramters are sorted by name while being serialized.
+Since this is really useful to provide better cache management and most servers don't really care about parameters order, this is the default behaviour.
```

Comment:
```suggestion
Since this is really useful to provide better cache management and most servers don't really care about parameter order, this is the default behaviour.
```